docs/content/overview/cdc-pipeline.md

```diff
@@ -11,6 +11,22 @@ CDC Streaming ELT Framework is a stream data integration framework that aims to
 * ✅ Synchronization of entire databases 
 * ✅ Schema evolution capability
 
+
+## Supported Connectors
+
+| Connector                                                | Database                                                                                                                                                                                                                                                                                                                                                                                               | Driver                    |
```

Comment:
Thanks for updating.
one left comment, maybe driver is unnecessary, because Doris & StarRocks is using sdk to load data, and we want to support Paimon/Iceberg/Kafka in the future, Driver is something used for jdbc connectors.

Reply:
I've fixed it.